lib/middleware/skip-workflow.js

```diff
@@ -17,6 +17,12 @@ module.exports = settings => {
       if (model === 'profile' && action === 'updateLastLogin') {
         return true;
       }
+      if (model === 'rop' && ['update', 'create', 'submit'].includes(action)) {
+        return true;
+      }
+      if (model === 'procedure' && ['update', 'create', 'delete'].includes(action)) {
+        return true;
+      }
```

Comment:
We probably want to make sure these still work at all if the action isn't one of those.

I'd suggest a small modification to the create hook to set all `rop` and `procedure` models to `autoresolved` as a fallback.

Reply:
not a bad shout